src/pages/credential-stuffing/credentialStuffing.module.scss

```diff
@@ -7,8 +7,8 @@
     margin-top: rem(24px);
   }
 
-  input.password {
-    display: inline;
+  .password {
+    margin-bottom: 0 !important;
```

Comment:
Are you sure that we need `!important` here?
May be we can increase selector specificity with other way?

Reply:
Good point, I made the selectors tag-specific and it seems to work, thanks! :)